gybitg/Models/Repositories/FakeAthleteRepository.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace gybitg.Models.Repositories
+{
+    public class FakeAthleteRepository : IAthleteRepository
+    {
+        //Is this workable?  Can I have these two things be populated by the seed data
+        // I have in another file?
+        public IQueryable<AthleteProfile> athleteProfiles => new List<AthleteProfile>
+        {
+            new AthleteProfile{}
```

Comment:
Do I need to somehow call the seed data from the other file?  Or should the seed data add the generated athletes to the correct repo?